components/organisms/NavBar.tsx

```diff
@@ -1,9 +1,14 @@
-import styled from 'styled-components';
+import React from 'react';
 
-export const MobileNavBar = styled.div`
+import MobileNavBar from './MobileNavbar';
+import DesktopNavbar from './DesktopNavbar';
+import { useViewport } from './ViewportProvider';
 
-`;
+const NavBar = () => {
+  const width = useViewport();
+  const breakpoint = 1200;
 
-const NavBar = () => <div>NAVBAR!</div>;
+  return (width !== undefined && width < breakpoint) ? <MobileNavBar /> : <DesktopNavbar />;
```

Comment:
Hiding it with a simple CSS media query would be much better, as this will break if there's no JS available on the Frontend.
(Note that we use Next.JS so that we have server-side rendering, where there is no information on the users' window size.)

Reply:
My idea was that the useEffect in the ViewportProvider ensures this only happens when it is mounted. Like here: https://dev.to/vvo/how-to-solve-window-is-not-defined-errors-in-react-and-next-js-5f97.

Granted, my solution differs a bit, but I saw the error you mentioned in my browser, and it disappeared like this.

Reply:
Yeah, the problem with that is, when the user does not have JS enabled (or when the user has a low-end device and JS performance is ~shit~ slow).

You could have both of them in the DOM, and simply chose which one to display like:

```css
.navbar-desktop { display: none;  }
.navbar-mobile  { display: block; }

@media screen and (min-width: 760px) {
    .navbar-desktop { display: block; }
    .navbar-mobile  { display: none;  }
}
```